bin/compile

```diff
@@ -651,3 +639,6 @@ echo "TOOL=${TOOL}" > "${t}"
 if [ "${TOOL}" != "gb" ]; then
     echo "NAME=${name}" >> "${t}"
 fi
+
+warn "We placed the compiled binary in /bin on your dyno."
```

Comment:
Is `warn` the correct command here? Do you intend for this to be yellow with exclamation marks? If you want it to be regulary text, you can use `echo`, otherwise 👍 

Could you attach a screenshot to the PR of this message in context?

Reply:
echo sounds great!